api/sbom.proto

```diff
@@ -144,6 +145,17 @@ message Tool {
     string vendor = 3;
 }
 
+message Lifecycle {
+    string name = 1;
+    optional string description = 2;
+    optional bool enum = 3;
+}
+
+message Identifier {
+    string type = 1;
+    string value = 2;
+}
```

Comment:
Please rebase the proto to the latest version. `Identifier` was removed in 22ba41acf26730dd3654cac0b6504a0f30a23476 .